src/components/input/LocationAddress.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <label
+    :class="{ focused: isInputFocused }"
+    v-overlay-container="{ togglePropName: 'displaySuggestions' }"
+    v-navigable-container="{ cursorPropName: 'cursorIndex' }"
+  >
+    <input
+      type="text"
+      @input="onInputInput"
+      @focus="onInputFocus"
+      @blur="onInputBlur"
+      @keydown="onInputKeyDown"
+      v-model="query"
+      :placeholder="placeholder"
+    />
+    <ul v-if="displaySuggestions">
+      <li
+        v-for="(suggestion, index) in suggestions"
+        :key="suggestion.id"
+        @click="onSuggestionClick(index)"
+        :class="{ cursor: index === cursorIndex }"
+      >
+        <span>{{ suggestion.address }}</span>
+      </li>
+    </ul>
+  </label>
+</template>
+<style scoped lang="scss">
+@import "../../style/variables";
+
+label {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  color: $greyscale-1;
+
+  &::after {
+    content: " ";
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 25%;
+    height: calc(100% - 2px);
+    background: linear-gradient(to right, rgba(255, 255, 255, 0), white);
+  }
+
+  &.focused::after {
+    content: none;
+  }
+}
+
+input {
+  font-size: 12px;
+  line-height: 1.83px;
+  border-width: 0 0 2px 0;
+  border-color: #4d4d4d;
+  color: #4d4d4d;
+  outline: none;
+  flex-grow: 1;
+}
+ul {
+  position: absolute;
+  top: 100%;
+  width: 100%;
+  border-width: 0;
```

Comment:
Can't lines 63 to 69 be replaced by `border: 0`?

Reply:
Fair point